features/step_definitions/assertion_steps.rb

```diff
@@ -0,0 +1,7 @@
+Then("I should see {string}") do |content|
+    expect(page).to have_content content
+end
+
+Then("I should be on the {string} page") do |content|
+    expect(page).to have_content content
```

Comment:
can you explain what is happening here?

Reply:
Done

Reply:
But you see what the step definition says?
You are supposed to test that the user should be on specific page

Reply:
Fixed